.buildkite/pipeline.yml

```diff
@@ -431,11 +431,23 @@ steps:
       - label: ":genie: EDMFX advection test in a box"
         command: >
           julia --color=yes --project=examples examples/hybrid/driver.jl --job_id edmfx_adv_test_box
-          --initial_condition DryAdiabaticProfileEDMFX --edmfx_adv_test true --turbconv edmfx --edmfx_entr_detr false
+          --initial_condition MoistAdiabaticProfileEDMFX --edmfx_adv_test true --turbconv edmfx --edmfx_entr_detr false
           --config box --hyperdiff true --kappa_4 1e8
-          --x_max 1e4 --y_max 1e4 --x_elem 2 --y_elem 2 --z_elem 45 --z_max 30e3 --dz_bottom 50
-          --moist equil --ode_algo SSP33ShuOsher --apply_limiter false --dt 0.1secs --t_end 1000secs
-          --dt_save_to_disk 10secs
+          --x_max 1e4 --y_max 1e4 --x_elem 2 --y_elem 2 --z_elem 45 --z_max 30e3 --dz_bottom 30
+          --moist equil --ode_algo SSP33ShuOsher --apply_limiter false --dt 10secs --t_end 1000secs
+          --dt_save_to_disk 100secs
+
+        artifact_paths: "edmfx_adv_test_box/*"
+        agents:
+          slurm_mem: 20GB
+      
+      - label: ":genie: EDMFX advection test in a sphere"
```

Comment:
@trontrytel I wonder if we want to keep both sphere and box, or replace the box with sphere?

Reply:
Lets keep both for now.